newIDE/app/src/EventsSheet/index.js

```diff
@@ -276,6 +276,18 @@ export default class EventsSheet extends React.Component<Props, State> {
     });
   };
 
+  _selectionCanReceiveSubEvents = () => {
```

Comment:
`_selectionCanHaveSubEvents` seems more similar to `canHaveSubEvents` of `event` :)

Reply:
:) good point, better follows the naming convention